### PR TITLE
Revert "Shaloo/ar 3945 readme"

### PR DESCRIPTION
**Pull request by shaloo**
Reverts arcana-network/demo-app-template#13

Add Ajith's feedback

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,26 +1,20 @@
...
 
-### Prerequisites
+## Quick Start
 
-- You'll need latest LTS versions of **`node.js`** and **`npm`** installed in your system for this code to work.
+- Prerequisites:
 
-- You must register and configure your dApp using [Arcana's Developer Dashboard](https://dashboard.beta.arcana.network). Configure social authentication mechanism, storage region, storage limits and other settings using the dashboard.
+  - You'll need latest LTS versions of **`node.js`** and **`npm`** installed in your system for this code to work.
+  - You'll need to register and configure your dApp using [Arcana's Developer Dashboard](https://dashboard.beta.arcana.network). Configure social authentication mechanism, storage region, storage limits and other settings using the dashboard.
```
Comment (shaloo) [inline README.md]:
```suggestion
You must register and configure your dApp using [Arcana's Developer Dashboard](https://dashboard.arcana.network). Configure the social authentication mechanism, storage region, storage limits, and other settings using the dashboard.

**Comment by shaloo** [comment]
Ajith, can u pls re-review dashboard link update fix. Thx

**Review by ajithranka — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -62,25 +56,3 @@ This project is built using [Vue 3](https://v3.vuejs.org/guide/introduction.html
 - [file-saver](https://www.npmjs.com/package/file-saver) to create and download files locally
 - [moment](https://www.npmjs.com/package/moment) for formatting date and time
 - [bytes](https://www.npmjs.com/package/bytes) for formatting bytes to human readable values
```
Comment (ajithranka) [inline README.md:58]: Usage, Support, Contributing Guide, and License sections are getting removed in this PR.

**Review by ajithranka — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (shaloo) [reply README.md:58]: Creating a fresh PR.  Is there a way to delete abandoned PRs?

**Pull request closed without merging**